Sources/DatadogCrashReporting/DDCrashReportingPlugin.swift

```diff
@@ -0,0 +1,35 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import Foundation
+import CrashReporter
+
+public class DDCrashReportingPlugin {
+    private static var sharedPLCrashReporter: PLCrashReporter?
+
+    public init?() {
+        DDCrashReportingPlugin.sharedPLCrashReporter = PLCrashReporter(
+            configuration: PLCrashReporterConfig(
+                signalHandlerType: .BSD,
+                symbolicationStrategy: .all
+            )
+        )
+    }
```

Comment:
i'm guessing there won't be this `public init?` in public API, at least can we put a `TODO: RUMM-956` for this method as well?

Reply:
There will be public `initializer` as discussed in RFC: Releasing iOS Crash Reporting. It won't be failable - I'm working on it right now in my next PR to `crash_reporting` branch.